amqp/src/test/java/akka/stream/alpakka/amqp/javadsl/AmqpConnectionProvidersTest.java

```diff
@@ -5,7 +5,11 @@
 package akka.stream.alpakka.amqp.javadsl;
 
 import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
[question] Why hamcrest's `assertThat`?

Reply:
The JUnit variant is deprecated nowadays.